scene/gui/control.cpp

```diff
@@ -421,7 +421,11 @@ void Control::_get_property_list(List<PropertyInfo> *p_list) const {
 				if (data.theme_constant_override.has(E.item_name)) {
 					usage |= PROPERTY_USAGE_STORAGE | PROPERTY_USAGE_CHECKED;
 				}
-				p_list->push_back(PropertyInfo(Variant::INT, PNAME("theme_override_constants") + String("/") + E.item_name, PROPERTY_HINT_RANGE, "-16384,16384", usage));
+				if (E.hint == PROPERTY_HINT_NONE) {
+					p_list->push_back(PropertyInfo(Variant::INT, PNAME("theme_override_constants") + String("/") + E.item_name, PROPERTY_HINT_RANGE, "-16384,16384", usage));
```

Comment:
Nothing to do here, but it's interesting that we limit item constants within `-2^14` and `2^14`. It's quite arbitrary. Perhaps it was done under the assumption that anything higher would break the the otherwise floating-point positions of Control nodes, as some constants are merely offsets.
Are values limited even in `set_theme_constant()` & co, I wonder...?

Reply:
Apart from beeing 32-bit it should not be limited by anything, so not sure why this range is used. If this is merged, it would make sense to check and set more specific ranges for existing constants.